app-benchmarks/stress-ng/files/stress-ng-0.09.31-makefile.patch

```diff
@@ -0,0 +1,80 @@
+--- a/Makefile	2018-06-21 15:39:27.000000000 +0200
++++ b/Makefile	2018-06-22 11:24:23.000000000 +0200
+@@ -21,7 +21,7 @@
+ # Codename "portable pressure producer"
+ #
+ 
+-CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -O2 -std=gnu99
++CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -std=gnu99
+ 
+ #
+ # Pedantic flags
+@@ -319,12 +319,10 @@
+ .o: stress-ng.h Makefile
+ 
+ .c.o: stress-ng.h Makefile $(SRC)
+-	@echo "CC $<"
+-	@$(CC) $(CFLAGS) -c -o $@ $<
++	$(CC) $(CFLAGS) -c -o $@ $<
+ 
+ stress-ng: info $(OBJS)
+-	@echo "LD $@"
+-	@$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
++	$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
+ 	@sync
+ 
+ .PHONY: info
+@@ -344,8 +342,7 @@
+ 		sed '$$ s/.$$//' >> apparmor-data.c
+ 	@echo "};" >> apparmor-data.c
+ 	@echo "const size_t g_apparmor_data_len = sizeof(g_apparmor_data);" >> apparmor-data.c
+-	@echo "CC $<"
+-	@$(CC) -c apparmor-data.c -o apparmor-data.o
++	$(CC) -c apparmor-data.c -o apparmor-data.o
+ 	@rm -rf apparmor-data.c apparmor-data.bin
+ 
+ #
+@@ -360,19 +357,14 @@
+ perf.o: perf.c perf-event.c
+ 	@$(CC) $(CFLAGS) -E perf-event.c | grep "PERF_COUNT" | sed 's/,/ /' | \
+ 	awk {'print "#define _SNG_" $$1 " (1)"'} > perf-event.h
+-	@echo CC $<
+-	@$(CC) $(CFLAGS) -c -o $@ $<
++	$(CC) $(CFLAGS) -c -o $@ $<
+ 
+ stress-vecmath.o: stress-vecmath.c
+-	@echo CC $<
+-	@$(CC) $(CFLAGS) -fno-builtin -c -o $@ $<
++	$(CC) $(CFLAGS) -fno-builtin -c -o $@ $<
+ 	@touch stress-ng.c
+ 
+ $(OBJS): stress-ng.h Makefile
+ 
+-stress-ng.1.gz: stress-ng.1
```

Comment:
You can skip this change to reduce patch size; if you aren't using the target, the rule won't be used either.

Reply:
Allright, dropped.